src/routes/api/search/[...path]/+server.ts

```diff
@@ -0,0 +1,36 @@
+import { dev } from '$app/environment';
+import getOptionalEnvVar from '$lib/utils/get-optional-env-var/private.js';
+import stripTrailingSlash from '$lib/utils/strip-trailing-slash.js';
+import { error } from '@sveltejs/kit';
+
+const MEILISEARCH_HOST = getOptionalEnvVar('MEILISEARCH_HOST');
+const MEILISEARCH_API_KEY = getOptionalEnvVar('MEILISEARCH_API_KEY');
+
+if (!dev && !MEILISEARCH_HOST) {
+  throw new Error('MEILISEARCH_HOST env var is required in production!');
+}
+
+export const POST = async ({ request, url }) => {
+  const path = url.pathname.replace('/api/search/', '');
+
+  if (!MEILISEARCH_HOST) return error(500, 'MEILISEARCH_HOST env var is required for search');
+
+  const result = await fetch(`${stripTrailingSlash(MEILISEARCH_HOST)}/${path}`, {
```

Comment:
You know what I'm going to write...lol
Check `result.ok`?

Reply:
don't think it's necessary, it just proxies the request and also forwards `.status` property. Up to the client-side code to interpret that (which it does by failing the parsing and displaying an error in the search results)